varia/sass/blocks/blog-posts/_style.scss

```diff
@@ -141,16 +148,16 @@
 					.has-background:not(.has-background-background-color) &,
 					[class*="background-color"]:not(.has-background-background-color) &,
 					[style*="background-color"] & {
-						color: currentColor;
+						color: currentcolor;
 					}
 				}
 			}
 		}
 	}
 }
 
-@import "../../base/extends";
```

Comment:
Removed `extends` import, because it creates duplicate styles.
It is already included here.
https://github.com/Automattic/themes/blob/18482fab16e744f723227652524bafa0ca8d2513/varia/sass/base/_imports.scss#L6-L8

Reply:
great catch!